enginetest/queries/create_table_queries.go

```diff
@@ -135,4 +135,15 @@ var CreateTableQueries = []WriteQueryTest{
 		SelectQuery:         "SHOW CREATE TABLE t1",
 		ExpectedSelect:      []sql.Row{sql.Row{"t1", "CREATE TABLE `t1` (\n  `pk` int NOT NULL,\n  `col1` blob DEFAULT (\"abc\"),\n  `col2` json DEFAULT (JSON_OBJECT(\"a\", 1)),\n  `col3` text DEFAULT (\"abc\"),\n  PRIMARY KEY (`pk`)\n) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_0900_bin"}},
 	},
+	{
+		WriteQuery: `CREATE TABLE td (
```

Comment:
What is this new test case covering?

Reply:
Does this cover anything beyond what the `show create table` test in `script_queries.go` below is already testing? Not bad to have this one located in `create_table_queries` I suppose, just making sure I wasn't missing anything else.

Reply:
Oh I forgot to check on this. I thought I didn't see literal default value defined for int and string types and use default value of string value for both types. I'll double check on this.